### ISSUE TITLE
neo4j-community_windows-x64_3_0_0-M02.exe Installation Issues

### ISSUE THREAD

**Issue by JKnowlesuk** [neo4j/neo4j#6202]
If you try to install the neo4j-community_windows-x64_3_0_0-M02.exe without the correct java version support it throws an error. 

Expected Behavior: Warn you are missing the required specific Java Version, or support more versions of java. 

Workaround: 
if you really intend to use the milestone release, try downloading the windows 64 bit offline version of java here first: http://www.java.com/en/download/manual.jsp
Java manual download page. Get the latest version of the Java Runtime Environment (JRE) for Windows, Mac, Solaris, and Linux.

As requested from Slack conversation in help-install

This was error log and my original Java version. 
[i4j_nlog_9.zip](https://github.com/neo4j/neo4j/files/87557/i4j_nlog_9.zip)
![java](https://cloud.githubusercontent.com/assets/1983193/12268937/e461e984-b946-11e5-8dcc-f951b3ef4752.png)

To get the 3_0_0-M02 community build to work you have to install.  The lastest specific Windows 64 as advised by Ryan below. 

"if you really intend to use the milestone release, try downloading the windows 64 bit offline version of java here first: http://www.java.com/en/download/manual.jsp
Java manual download page. Get the latest version of the Java Runtime Environment (JRE) for Windows, Mac, Solaris, and Linux."

"
ryguyrg.neo [9:00 PM] 
not sure yet why the installer isn’t working for the 3.0 M02 release

if you really intend to use the milestone release, try downloading the windows 64 bit offline version of java here first: http://www.java.com/en/download/manual.jsp
Java manual download page. Get the latest version of the Java Runtime Environment (JRE) for Windows, Mac, Solaris, and Linux.

ryguyrg.neo [9:02 PM] 
you have to explicitly find the 64 bit version — oracle only gives you the 32 bit through the normal process"

**Comment by spacecowboy** [comment]
@JamesKnowlesEbsta great that you're trying out 3.0! Just to confirm that I understand the problem correctly,
- you previously had installed a 32-bit JRE
- tried running the 64-bit version of Neo4j
- at which point Neo4j crashed with an error

and
- in your opinion, the error message was not sufficiently helpful?

Would it be enough if the error message said something like

```
Error: This is a 64bit version of Neo4j and you are trying to run it with 32bit java. Re-run on 64bit java.
```

instead? (Not entirely sure what the error message actually is atm)

**Comment by benbc** [comment]
@pgpv Can you post an update here? Is this fixed in 3.0.0-M03?

**Comment by jjaderberg** [comment]
No response, closing.
